### PR TITLE
Run multiple engines integration test.

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: .github/workflows/continuous.yml [hunk 1]
```diff
@@ -145,6 +145,7 @@ jobs:
       - name: Compile PaddlePaddle JNI
         run: ./gradlew :paddlepaddle:paddlepaddle-native:compileJNI
       - name: Integration tests
+        # MXNet integration test on Github Action fail due to memory limitation on Windows
         run: |
-          ./gradlew :paddlepaddle:paddlepaddle-engine:test :paddlepaddle:paddlepaddle-model-zoo:test
```
Comment (lanking520) [inline .github/workflows/continuous.yml:149]: This will disable all paddle and onnx test coverage.

File: .github/workflows/continuous.yml [hunk 2]
```diff
@@ -145,6 +145,7 @@ jobs:
...
-          ./gradlew :onnxruntime:onnxruntime-engine:test
+          ./gradlew --rerun-tasks -Dai.djl.default_engine=PyTorch build
```
Comment (lanking520) [inline .github/workflows/continuous.yml:150]: Do rerun on all tasks will make the test time tripled. I would still suggest we do test in parallel and for the sake of clearance.

**Review by lanking520 — CHANGES_REQUESTED** [review]

File: .github/workflows/continuous.yml (hunk 1 above)
Comment (frankfliu) [reply .github/workflows/continuous.yml:149]: 1. We are not collecting test coverage here, this run doesn't runt jRR
2. gradle build will run all tests except MXNet integration test

File: .github/workflows/continuous.yml (hunk 2 above)
Comment (frankfliu) [reply .github/workflows/continuous.yml:150]: 1. The build itself is very fast, less than 1 minute, especially for TF and PT integration test
2. The majority time spend is on spin up the machine, download engine native binary and dependencies. 
3. Windows build is run parallel with Linux and mac, it should not increase total continuous build time

**Review by roywei — APPROVED** [review]

**Review by lanking520 — APPROVED** [review]

**Pull request merged**